src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -939,6 +939,8 @@ describe('toolbar mode only features', () => {
 
       wrapper.findDrawerTriggerById('global-drawer-1')!.click();
 
+      await delay();
```

Comment:
the test does not work without this condition anymore, because now setting `openDrawersHistory` works asynchronously